tractor/_ipc.py

```diff
@@ -42,25 +43,25 @@ def __init__(self, stream):
                 yield packet
 
     @property
-    def laddr(self):
+    def laddr(self) -> Tuple[str, int]:
         return self._laddr
 
     @property
-    def raddr(self):
+    def raddr(self) -> Tuple[str, int]:
         return self._raddr
 
-    async def put(self, data):
+    async def put(self, data: Any) -> int:
         async with self._send_lock:
             return await self.stream.send_all(
                 msgpack.dumps(data, use_bin_type=True))
 
-    async def get(self):
+    async def get(self) -> Any:
```

Comment:
Might want to look and see if you can define a `TypeVar` for this instead: https://mypy.readthedocs.io/en/latest/generics.html

```python
T = TypeVar('T')

class StreamQueue(Generic[T]):
    async def get(self) -> Any:
        pass
```

You can get a little bit more type safety this way if you know what the return type will be situationaly:

```python
queue = StreamQueue[str](steam)
```

For a `StreamQueue` that operates over strings.